lib/YaST/Bootloader/BootloaderSettingsController.pm

```diff
@@ -79,6 +88,13 @@ sub write_to_partition {
     $self->get_boot_code_options_page->press_next();
 }
 
+sub disable_grub_timeout_navigating_tabs {
+    my ($self) = @_;
+    $self->get_bootloader_options_Tab->use_navigating_tabs();
```

Comment:
your POM should represent some part of the screen, not just one Tab, plus should be in lower letter.
IMO the getter should be: `get_bootloader_settings_tabs()` (please, do not forget the `()` for each function to follow the current coding style).
And the function will be just `navigate()`,
so `$self->get_bootloader_settings_tab->navigate()`.